coordinator/src/main/java/org/apache/uniffle/coordinator/SimpleClusterManager.java

```diff
@@ -205,21 +215,74 @@ private void updateExcludeNodes(String path) {
     CoordinatorMetrics.gaugeExcludeServerNum.set(excludeNodes.size());
   }
 
-  private void parseExcludeNodesFile(DataInputStream fsDataInputStream) throws IOException {
+  private Set<String> parseExcludeNodesFile(
```

Comment:
Please change all the places of "exclude node" or "excludeNode".

Reply:
> Please change all the places of "exclude node" or "excludeNode".

done.